features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/SelectAppPanel.jsx

```diff
@@ -57,7 +57,7 @@ const SelectAppPanel = (props) => {
             <Grid x={12} md={6} className={classes.centerItems}>
                 <TextField
                     fullWidth
-                    id='outlined-select-currency'
+                    id='selected-application'
```

Comment:
I think we don't need IDs here and below as well , can remove them

Reply:
When we add the ID it generates the ` aria-describedby ` for accessibility